__sdk__.js

```diff
@@ -6,5 +6,10 @@ module.exports = {
         entry:           './src/index',
         staticNamespace: '__muse__',
         automatic:        true
+    },
+    'tracker': {
+        entry:           './src/tracker',
+        staticNamespace: '__tracker__',
+        automatic:        true
```

Comment:
Can we remove `automatic: true`? This is a new integration pattern, so merchants should pass `components=buttons,tracker` when integrating this. That way they'll only get the specific code they need.